<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PhysicLab</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0,0,0">
    <item>
     <widget class="QPushButton" name="newFigButton">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="text">
       <string>新建图片</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MatplotlibWidget" name="canvas" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="rowWrapPolicy">
       <enum>QFormLayout::DontWrapRows</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="LegendLabel">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>图例文字</string>
        </property>
        <property name="textFormat">
         <enum>Qt::MarkdownText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LegendEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="getXButton">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="text">
         <string>添加X轴数据</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="getXLineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="getYButton">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="text">
         <string>添加Y轴数据</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="getYLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="markerLabel">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>选择散点图标记</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="markerComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>7</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 7pt &quot;微软雅黑&quot;;</string>
        </property>
        <item>
         <property name="text">
          <string>圆圈（默认）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X型</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>朝上的三角</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>不加散点</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="loadExcelButton">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="text">
       <string>导入excel数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addLineButton">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="text">
       <string>绘图</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveFigButton">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="text">
       <string>保存图片</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>addLineButton</zorder>
   <zorder>canvas</zorder>
   <zorder>loadExcelButton</zorder>
   <zorder>saveFigButton</zorder>
   <zorder>newFigButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>482</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatplotlibWidget</class>
   <extends>QWidget</extends>
   <header>app.View.Widgets.MatplotlibWidget</header>
   <container>1</container>
   <slots>
    <signal>figChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <signal>newFigCreated(Figure)</signal>
 </slots>
</ui>
